### PR TITLE
8 domains

### LINKED ISSUE #3895: marketscreener.com
### Basic Information

**Test link:** `https://www.marketscreener.com/`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.145

### Additional Message

placeholders

### LINKED ISSUE #3915: adage.com
### Basic Information

**Test link:** `https://adage.com/article/agency-news/tiktok-hires-rpa-and-initiative-ramp-us-marketing-efforts/2181191`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.145

### Additional Message

placeholders

### LINKED ISSUE #4026: test.abovetopsecret.com
### Basic Information

**Test link:** `http://test.abovetopsecret.com/forum/thread1212355/pg1`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.148

### Additional Message

placeholders

### LINKED ISSUE #4066: streamingmedia.com
### Basic Information

**Test link:** `https://www.streamingmedia.com/Articles/ReadArticle.aspx?ArticleID=135703`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.149

### Additional Message

big placeholder appears on first visit

### LINKED ISSUE #4080: tellymix.co.uk
### Basic Information

**Test link:** `https://tellymix.co.uk/tv/444985-ant-and-dec-apologise-for-old-episodes-of-saturday-night-takeaway-featuring-blackface.html`
**Category:** other
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.149

### Additional Message

placeholders
`tellymix.co.uk##.adb_top`

### LINKED ISSUE #4266: fxempire.com
### Basic Information

**Test link:** `https://www.fxempire.com/news/article/china-blue-chip-index-hits-5-year-high-australian-shares-tumble-as-borders-close-659419`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.150

### Additional Message

```
||engine.fxempire.com^
||cdn.fxempire.com^
```

CNAME blocking takes care of it in Firefox, still there are leftover placeholders though.

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Fix https://github.com/NanoMeow/QuickReports/issues/3895
* Fix https://github.com/NanoMeow/QuickReports/issues/4080
* Fix https://github.com/NanoMeow/QuickReports/issues/4066
* Fix https://github.com/NanoMeow/QuickReports/issues/4026
* Fix https://github.com/NanoMeow/QuickReports/issues/3915
* Fix https://github.com/NanoMeow/QuickReports/issues/4266

Example pages:
```
! International
! Fix(?) https://github.com/NanoMeow/QuickReports/issues/3895
! https://www.marketscreener.com/
marketscreener.com##.myFooter
marketscreener.com##[id^="zpp"]
! Fix https://github.com/NanoMeow/QuickReports/issues/4266#issuecomment-670030414
! https://www.fxempire.com/news/article/china-blue-chip-index-hits-5-year-high-australian-shares-tumble-as-borders-close-659419
||engine.fxempire.com^
||cdn.fxempire.com^
fxempire.com##button:has-text(/Sponsored|Advertising Disclosure/):has(.fa-info-circle):upward([class*="fx-grid"])
fxempire.com##span[id^="zone"]:empty:upward([class*="-sc-"][class*="__"]>div)
fxempire.com##[class*="TopButtonAd"]
fxempire.com##li[class^="Tab-"] div[class*=" fx-grid__ColSystem-"]:has-text(Top Brokers)
! https://en.glosbe.com/nb/it/svada
glosbe.com###topContainer

! English
! http://www.joshuawright.net/slack-wyrm-642.html
joshuawright.net##.adsbygoogle:upward(1)
! Fix https://github.com/NanoMeow/QuickReports/issues/4080
! https://tellymix.co.uk/tv/444985-ant-and-dec-apologise-for-old-episodes-of-saturday-night-takeaway-featuring-blackface.html
tellymix.co.uk##.adb_top
tellymix.co.uk##.a-text
! Fix https://github.com/NanoMeow/QuickReports/issues/4066
! https://www.streamingmedia.com/Articles/ReadArticle.aspx?ArticleID=135703
streamingmedia.com##.container1
! Fix https://github.com/NanoMeow/QuickReports/issues/4026
! http://test.abovetopsecret.com/forum/thread1212355/pg1
abovetopsecret.com###third300
abovetopsecret.com##.threadpost > [style^="width: 630px;"]
! Fix https://github.com/NanoMeow/QuickReports/issues/3915
! https://adage.com/article/agency-news/tiktok-hires-rpa-and-initiative-ramp-us-marketing-efforts/2181191
adage.com##.ad-entity-container
adage.com###piano-inline_nsl
```

I also removed `spankwire.com###js-react-video-player div:has-text(/^Advertisement$/):upward(2)`, which seemed to me to hide the actual video on `https://www.spankwire.com/Hooded-latex-lesbian-lovers/video121301/`.

### CODE REVIEW

**Comment by jspenguin2017** [comment]
My desktop computer is broken, and I don't have the linting tool set up on my laptop, so this can take some time. I hope I can have time to get my computer repaired this weekend, if not, I'll get it fixed by next weekend for sure.

**Comment by DandelionSprout** [comment]
Duly noted. Wishing the best for your desktop PC.

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -294,6 +294,21 @@ lmtonline.com###menuBar, #subHead:style(transform: none; -webkit-transform: none
 lmtonline.com###menuBar:style(margin-top: 50px !important;)
 lmtonline.com##.ctpl-fullbanner
 
+# https://github.com/NanoMeow/QuickReports/issues/3895
+marketscreener.com##.myFooter
+marketscreener.com##[id^="zpp"]
+
+# https://github.com/NanoMeow/QuickReports/issues/4266
+||engine.fxempire.com^
+||cdn.fxempire.com^
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt]: I think these should be in EasyList.

**Comment by DandelionSprout** [comment]
`||engine.fxempire.com^` seems to be in AdGuard Base. I can't seem to reproduce `||cdn.fxempire.com^`. So I guess I can remove those two entries.

File: dist/placeholder-buster.txt (hunk 1 above)
Comment (DandelionSprout) [reply dist/placeholder-buster.txt]: I've removed those two entries now, as explained below.

**Comment by jspenguin2017** [comment]
`https://en.glosbe.com/nb/it/svada`

**Comment by jspenguin2017** [comment]
`http://www.joshuawright.net/slack-wyrm-642.html`

**Pull request merged**